src/styles/blocks/card.scss

```
@@ -0,0 +1,81 @@
+html {
+  font-family: Roboto, Helvectica, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $boarder-color;
+  color: $main-text-color;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  box-shadow: 1px 1px 2px $box-shadow;
+}
```

why don't you use parrent selector?
https://sass-lang.com/documentation/style-rules/parent-selector/
```suggestion
 &__image {},
 &__name {},
 &__code {},
 ...
}
```